sdk/ai/azure-ai-projects/samples/agents/sample_agents_stream_with_base_override_eventhandler.py

```diff
@@ -0,0 +1,96 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+FILE: sample_agents_stream_with_base_override_eventhandler.py
+
+DESCRIPTION:
+    This sample demonstrates how to override the base event handler and parse the events and iterate through them
```

Comment:
Would be maybe good to add more information about when it is useful to override

Reply:
Done